conf/defaults.ini

```diff
@@ -307,6 +307,9 @@ oauth_state_cookie_max_age = 600
 # limit of api_key seconds to live before expiration
 api_key_max_seconds_to_live = -1
 
+# Set to true to enable SigV4 authentication option for HTTP-based datasources
```

Comment:
As discussed we may want to move this to another section, but lets consider that in follow up PR.